OpenUpSA/wazimap-ng-ui
# positioning points by their anchor instead of their center

## Description
previously we didn't set the `iconAnchor` property while creating the markers. this let the markers being positioned to the left of the actual coordinates. 

https://stackoverflow.com/questions/46101450/explanation-of-leaflet-custom-icon-latlng-vs-xy-coordinates


## Related Issue
https://wazimap.atlassian.net/browse/WNCM-108


## How is it tested automatically?
this is not tested automatically

## How should a reviewer test it locally
* go to Sifar profile
* using the point mapper select `General Practitioners`
* confirm that the marker of the western-most point(Dr AJ (Andre John) Stoffels) is not placed in the ocean and it's position does not change as you zoom in


## Screenshots
* before the fix
![image](https://user-images.githubusercontent.com/53019884/138571536-4b98ce56-d6aa-400a-879a-f8ba956d416c.png)

* after the fix
![image](https://user-images.githubusercontent.com/53019884/138571547-6217868f-a4a1-4971-aaf2-7e572ff1210a.png)



## Changelog

### Added

### Updated
* `point_data.js` to set the `iconAnchor` while creating the markers

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
